src/components/ContextMenu.vue

```diff
@@ -25,9 +25,11 @@ export default {
   },
   updated: function () {
     this.setContainerPosition(this.$store.state.events.contextMenu.event);
+
   },
   methods: {
     setContainerPosition(event) {
+      if (event !== null) {
```

Comment:
If you think about this, not having an event and trying to display the context menu will still produce a bug in the app. Skipping this code block with an if statement will only get rid of the error in the console which could actually make it harder to debug the issue.

We could keep it, just a school of thought I wanted to share.

Also, the indentation is off. (Ctrl + Shift + F)